src/styles/List.js

```diff
@@ -7,19 +7,12 @@ const Container = styled.ul`
   margin: 0;
 
   li {
-    padding: 16px;
+    padding: 20px;
+    border-top: 1px solid #e1e4e8;
   }
 
   li:hover {
     color: ${colors.lightGrey};
-    background-color: ${colors.lighterGrey};
-  }
-
-  li:nth-child(even):hover {
-    background-color: ${colors.lighterGrey};
-  }
-
-  li:nth-child(even) {
```

Comment:
Why delete this? My preference is towards having visual separation leverage grey.

Reply:
@bdougie Ok, let me roll back and test it!

This was inspired by GitHub's issue list.

Reply:
No need to revert. I didn't grasp the content. Perhaps we can make the issues title grey instead? This will prevent the whole from the page being filled with white boxes.

Let's see this through. Can you make the height of the card slimmer as well? To align with the github slyle

Reply:
Sure I will make it @bdougie!